### DISCUSSION TITLE
Code to port

### DISCUSSION THREAD

**Discussion by manchesterfamily** [unikraft/unikraft#569] [General]
Hi, 

I am looking to port this code in unikraft, any help would be appreciated in this regards.

import os,sys
from PIL import Image, ImageFilter

for infile in sys.argv[1:]:
          f, e = os.path.splitext(infile)
          outfile = f+".jpg"
          if infile != outfile:
                   try:
                               with Image.open(infile) as im:
                                             im_grey = im.convert("L")
                                             im_grey.save(outfile)
                    except OSError:
                               print("cannot convert",infile)

Regards

**Comment by razvand** [comment]
@manchesterfamily, since we are already (viciously) using [the Unikraft Discord server](https://bit.ly/UnikraftDiscord) for discussions, meetings, brainstorming and jokes (mostly bad, we do what we can), we will close the Discussions tab. We will do this on Friday, March 31, 2023.

Please move any discussions or questions on [the Unikraft Discord server](https://bit.ly/UnikraftDiscord). See you all there! :handshake:
